changelogs/fragments/depricated_next_hop_self_bgpAddFamilt.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Deprecated next_hop_self type bool and introduced nexthop_self as dict under bgp_address_family.
```

Comment:
If this also fixes the `soft reconfiguration` issue, please mention it under `bugfixes`.

Reply:
@NilashishC  Thanks